controllers/yawol-controller/loadbalancer/loadbalancer_controller.go

```diff
@@ -1117,6 +1117,24 @@ func (r *Reconciler) deleteFips(
 
 	var requeue = false
 
+	// skip deletion and release status when annotated
+	if lb.GetAnnotations()[yawolv1beta1.LoadBalancerKeepFloatingIP] == "true" {
```

Comment:
I found one more thing.
Normally we use `strconv.ParseBool()` for boolean annotations (example `yawolv1beta1.LoadBalancerAdHocDebug`)